coral-component-shell/src/styles/index.styl

```diff
@@ -28,7 +28,7 @@ coral-shell-content {
   right: 0;
 
   -webkit-overflow-scrolling: touch;
-  overflow: auto;
+  overflow: hidden;
```

Comment:
it is not a good approach to limit overflow. Due to such limitation user won't be able to scroll both horizontally and vertically. In case there are many pages author won't be able to scroll vertically to reach those on the bottom of the list. I suggest getting only overflow-x or the best would be to determine why the page is not suited for 320px and fix the original issue with some styling (like `max-width: 100%`) instead of using `overflow` property.